imports/ui/Pages/Admin/ShowScreen/index.js

```diff
@@ -26,47 +31,74 @@ const propTypes = {
-      {isReady &&
-        interactions.map((i) => (
-          <ul key={i._id}>
-            <li>
-              {i.state === 'ACTIVE' && 'active:'} {i.type}{' '}
-              <button onClick={() => startInteraction.call({ interactionId: i._id })}>Start</button>
-            </li>
-          </ul>
-        ))}
-    </div>
-    <div>
-      <button disabled={!hasPrevious} onClick={() => previousInteraction.call()}>
-        Previous
-      </button>
-      <button disabled={!hasNext} onClick={() => nextInteraction.call()}>
-        Next
-      </button>
-    </div>
-    <div>
-      {isReady && (
-        <UpdateGames
-          games={games}
-          updateScores={({ _id: id, pointsCandidate1, pointsCandidate2 }) =>
-            updateInteractionDetails.call({ id, data: { pointsCandidate1, pointsCandidate2 } })
-          }
-          setWinner={({ _id: id, winner }) =>
-            updateInteractionDetails.call({ id, data: { winner } })
-          }
-        />
-      )}
-    </div>
-  </AdminLayout>
-);
+const ShowScreen = ({ classes, isReady, interactions, games, hasNext, hasPrevious }) => {
+
+  return (
+    <AdminLayout>
+      <div>
+        {!isReady && <div>Loadings</div>}
+        {isReady &&
+          interactions.map((i) => (
+            <ul key={i._id}>
+              <li>
+                {i.state === 'ACTIVE' && 'active:'} {i.type}{' '}
+                <Button
+                  variant="contained"
+                  size="large"
+                  className={classNames(classes.margin)}
+                  onClick={() => startInteraction.call({ interactionId: i._id })}
+                >
+                  Start
+                </Button>
+              </li>
+            </ul>
+          ))}
+      </div>
+      <div>
+        <Button
+          variant="contained"
+          size="large"
+          className={classNames(classes.margin)}
+          disabled={!hasPrevious} onClick={() => previousInteraction.call()}
+        >
+          Previous
+        </Button>
+        <Button
+          variant="contained"
+          size="large"
+          className={classNames(classes.margin)}
+          disabled={!hasNext} onClick={() => nextInteraction.call()}
+        >
+          Next
+        </Button>
+      </div>
+      <div>
+        {isReady && (
+          <UpdateGames
+            games={games}
+            updateScores={({ _id: id, pointsCandidate1, pointsCandidate2 }) =>
+              updateInteractionDetails.call({ id, data: { pointsCandidate1, pointsCandidate2 } })
+            }
+            setWinner={({ _id: id, winner }) =>
+              updateInteractionDetails.call({ id, data: { winner } })
+            }
+          />
+        )}
+      </div>
+    </AdminLayout>
+  );
+}
+
+const styles = theme => ({
+  margin: {
+    margin: theme.spacing.unit,
+  }
+});
+
 
 ShowScreen.propTypes = propTypes;
 
-export default withTracker(() => {
+export default withStyles(styles)(withTracker(() => {
```

Comment:
Funktioniert das sorum? Bei mir hat das immer einen Fehler geworfen. Lösung war, das `withStyles(styles)(Component)` in die returnte Funktion von `withTracker` zu übergeben